release-1.9/release-1.9.md

```diff
@@ -0,0 +1,100 @@
+# Proposed timeline for 1.9
+We will begin the 1.9 release the first week of October 2017.
+
+The proposal below is similar in layout previous releases:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+- there are a number of North American holidays during Q4, so this release will
+  have reduced scope and fewer features
+- same as in 1.8
+  * alpha's should not be cut unless all master-release-blocking tests are passing
+  * release notes for planned features done 2 weeks out from feature freeze
+  * docs PRs deadlines ahead of release
+
+## 1.9 Release Schedule Key Dates
+- Oct 5nd (Weds) coding start
```

Comment:
should be "5th", isn't it ? and 2017-10-05 is Thursday.

Reply:
thx